MyLibrary/Tests/ScheduleFeatureTests/Mocks.swift

```diff
@@ -95,3 +97,28 @@ extension Speaker {
     ]
   )
 }
+
+extension Favorites {
```

Comment:
You can add mock under the your test code.

Reply:
I moved these specific mocks near related tests in 4e279e4c9ccb3c9114c3af9533abb767731574fe.